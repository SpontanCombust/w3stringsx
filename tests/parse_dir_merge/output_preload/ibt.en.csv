;idspace=????
;section=menu
panel_Mods|panel_Mods
panel_ibt_name|panel_ibt_name
preset_Mods_ibt_name|preset_Mods_ibt_name
preset_value_ibt_default|preset_value_ibt_default
option_ibt_hair_short_tied|option_ibt_hair_short_tied
option_ibt_hair_short_untied|option_ibt_hair_short_untied
preset_value_ibt_hairstyle_short_loose|preset_value_ibt_hairstyle_short_loose
preset_value_ibt_hairstyle_elven_rebel|preset_value_ibt_hairstyle_elven_rebel
;section=bundle
item_desc_ibt_hairtie|item_desc_ibt_hairtie
item_desc_ibt_scissors|item_desc_ibt_scissors
item_desc_ibt_tonic_beard|item_desc_ibt_tonic_beard
item_desc_ibt_tonic_hair|item_desc_ibt_tonic_hair
item_name_ibt_hairtie|item_name_ibt_hairtie
item_name_ibt_scissors|item_name_ibt_scissors
item_name_ibt_tonic_beard|item_name_ibt_tonic_beard
item_name_ibt_tonic_hair|item_name_ibt_tonic_hair
item_name_recipe_ibt_tonic_beard|item_name_recipe_ibt_tonic_beard