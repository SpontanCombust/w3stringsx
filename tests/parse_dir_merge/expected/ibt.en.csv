;idspace=????
;section=menu
panel_Mods|panel_Mods
panel_ibt_name|panel_ibt_name
preset_Mods_ibt_name|preset_Mods_ibt_name
preset_value_ibt_default|preset_value_ibt_default
option_ibt_hair_short_tied|option_ibt_hair_short_tied
option_ibt_hair_short_untied|option_ibt_hair_short_untied
preset_value_ibt_hairstyle_short_loose|preset_value_ibt_hairstyle_short_loose
preset_value_ibt_hairstyle_elven_rebel|preset_value_ibt_hairstyle_elven_rebel
preset_value_ibt_hairstyle_shaved_with_ponytail|MISSING_LOCALISATION
preset_value_ibt_hairstyle_mohawk_with_ponytail|MISSING_LOCALISATION
;section=bundle
item_desc_ibt_hairtie|item_desc_ibt_hairtie
item_desc_ibt_scissors|item_desc_ibt_scissors
item_desc_ibt_tonic_beard|item_desc_ibt_tonic_beard
item_desc_ibt_tonic_hair|item_desc_ibt_tonic_hair
item_name_ibt_hairtie|item_name_ibt_hairtie
item_name_ibt_scissors|item_name_ibt_scissors
item_name_ibt_tonic_beard|item_name_ibt_tonic_beard
item_name_ibt_tonic_hair|item_name_ibt_tonic_hair
item_name_recipe_ibt_tonic_beard|item_name_recipe_ibt_tonic_beard
;section=scripts
ibt_notif_drank_tonic_beard_failure|MISSING_LOCALISATION
ibt_notif_drank_tonic_beard_success|MISSING_LOCALISATION
ibt_notif_drank_tonic_hair_failure|MISSING_LOCALISATION
ibt_notif_drank_tonic_hair_success|MISSING_LOCALISATION
ibt_notif_used_scissors_beard_failure|MISSING_LOCALISATION
ibt_notif_used_scissors_beard_success|MISSING_LOCALISATION
ibt_notif_used_scissors_hair_failure|MISSING_LOCALISATION
ibt_notif_used_scissors_hair_success|MISSING_LOCALISATION
item_desc_ibt_scissors_mode_beard|MISSING_LOCALISATION
item_desc_ibt_scissors_mode_hair|MISSING_LOCALISATION
item_desc_ibt_scissors_mode_preamble|MISSING_LOCALISATION